src/app/scheduling/datepicker/reactive-form/reactive-form.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="date-picker-wrapper">
+    <form [formGroup]="dateTimeForm">
+        <div class="form-group">
+            <div class="form-date-picker">
+                <igx-date-picker formControlName="date" [value]="date" [minValue]="minDate" [maxValue]="maxDate">
+                    <label igxLabel>Start Date</label>
```

Comment:
Saying `Start Date` and `Start Time` implies that there will be `End Date`/`End Time`, maybe just `Date` and `Time` for labels?